src/main/java/com/kucoin/sdk/rest/response/CurrencyResponse.java

```diff
@@ -23,11 +24,12 @@
 
     private int precision;
 
+    private BigDecimal withdrawalMinSize;
+
     private BigDecimal withdrawalMinFee;
 
-    private BigDecimal withdrawalMinSize;
+    private Boolean isWithdrawEnabled;
 
-    private boolean isIsWithdrawEnabled;
+    private Boolean isDepositEnabled;
```

Comment:
These two changes break deserialization.

The field is called `isDepositEnabled` in the JSON, which Jackson attempts to map to a property called `depositEnabled`, not `isDepositEnabled`.  In the previous code I just called the property `isIsDepositEnabled`, which is ugly.

The right answer is:

```
@JsonProperty("isDepositEnabled")
private Boolean isDepositEnabed;
```
I believe.

The same applies to `isWithdrawEnabed`.

Reply:
You're right. I'll fix it right away. Thanks